3.10/installation-windows.md

```diff
@@ -17,19 +18,30 @@ Visit the official [Download](https://www.arangodb.com/download){:target="_blank
 page of the ArangoDB web site.
 
 You may verify the download by comparing the SHA256 hash listed on the website
-to the hash of the file. For example, you can you run `openssl sha256 <filename>`
+to the hash of the file. For example, you can run `openssl sha256 <filename>`
 or `certutil -hashfile <filename> sha256` in a terminal.
 
 {% hint 'info' %}
 Running production environments on Windows is not supported.
 {% endhint %}
 
+Docker
+------
+
+The recommended way of using ArangoDB on Windows is to use the
+[ArangoDB Official Docker images](https://hub.docker.com/_/arangodb){:target="_blank"}
+with, for instance, [Docker Desktop](https://www.docker.com/products/docker-desktop/){:target="_blank"}. You can also use the [Docker images](https://hub.docker.com/r/arangodb/arangodb){:target="_blank"} that are directly published by ArangoDB on a regular basis.
```

Comment:
I think it would be clearer if we mentioned the image names, `arangodb` and `arangodb/arangodb` here, so that it's easier to distinguish the links.